test/orm/declarative/test_tm_future_annotations.py

```diff
@@ -377,7 +374,7 @@ class A(decl_base):
             id: Mapped[int] = mapped_column(primary_key=True)
             data: Mapped[str] = mapped_column()
 
-            bs: Mapped[KeyFuncDict[str, "B"]] = relationship(  # noqa: F821
```

Comment:
this test is explicitly "literal_annotations" (note the name), this can't change